src/policy/largeobjectspace.rs

```diff
@@ -95,6 +83,41 @@ impl<VM: VMBinding> Space<VM> for LargeObjectSpace<VM> {
 }
 
 impl<VM: VMBinding> LargeObjectSpace<VM> {
+    #[cfg(target_pointer_width = "64")]
+    const MARK_TABLE: SideMetadataSpec = SideMetadataSpec {
```

Comment:
Is there any reason for moving  mark-bit and nursery-bit to side metadata?

Reply:
Because header mark bits are not supported on openjdk. Currently gc byte on openjdk is in header as well and they cannot be kept after gc(due to the limitations of openjdk)